operations.go

```diff
@@ -36,20 +36,20 @@ type OperationResponse struct {
 	Op Operation `json:"operation"`
 }
 
-type OperationRunOneOffData struct {
+type OperationStartOneOffData struct {
 	AttachURL   string            `json:"attach_url"`
 	ContainerID map[string]string `json:"container_id"`
```

Comment:
Why is the `ContainerID` a map?

Reply:
When trying to set it as a string, it can't be unmarshalled.
Maybe the return of the api is not the right one? :thinking: 
Here is the related PR: https://github.com/Scalingo/api/pull/2147
Could you have a look at it @Soulou ? :pray:

Reply:
Yep the return of the API is probably not the right one. you must be serializing a BSON::ObjectId, leading to `{$oid: "12345"}` in JSON instead of `12345`